docs/apis/plugintypes/ai/provider.md

```diff
@@ -5,26 +5,34 @@ tags:
   - LLM
   - Provider
 ---
-Providers are the interface between the LMS AI subsystem and external AI systems.
-Their focus should be on converting the data requested by an Action into the format needed
-by the external AI services API, and then correctly providing the response back from the AI
-in an Action Response object.
+
+Providers are the interface between the [AI subsystem](/apis/subsystems/ai/index.md) and external AI.
+Their focus should be on converting the data requested into the format needed
+by the external AI, and then correctly providing the response back.
+
+Incoming data to the Provider plugin arrives via the Manager `core_ai\manager`.
+The Manager is the connective tissue between the Provider and the [Placement](/apis/plugintypes/ai/placement.md) plugins.
+Likewise, all responses from the Provider plugin are handed back to the Manager before being passed to the Placement plugin.
 
 :::warning The Golden Rule:
 
-Placements DO NOT know about Providers, and Providers DO NOT know about Placements.
+Placements **do not** know about Providers, and Providers **do not** know about Placements.
 Everything should go via the Manager.
 
 :::
 
+## Class implementation
+
 Providers are defined as classes in their own namespace according to their plugin name.
-The naming convention for Action classes is `aiprovider_<plugin name>`,
-for example: `aiprovider_openai`, `aiprovider_azureai`. With corresponding namespaces.
+The naming convention for a Provider class is `aiplacement_<plugin name>`.
+For example: `aiplacement_openai`, or `aiprovider_azureai` (with a corresponding namespace).
```

Comment:
There are two small typos here, referencing **aiplacement_** when it should be **aiprovider_**

Reply:
Thanks Mick. Good catch. I have fixed those up for 4.5 and 5.0